lib/takeaway.rb

```diff
@@ -0,0 +1,32 @@
+# deals with orders + cost
+require 'order'
+require 'sms'
+
+class Takeaway
+
+	def initialize(menu:, config:, order: nil, sms: nil) #dependency injection - 'other class is injected into this class to enable menu class methods to be accessed'
```

Comment:
Metrics/LineLength: Line is too long. [163/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)